classes/app.rb

```diff
@@ -0,0 +1,138 @@
+require_relative 'person'
+require_relative 'decorator'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'classroom'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @classrooms = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'There are no books in the library'
+    else
+      @books.each do |book|
+        puts "Title: #{book.title}, Author: #{book.author}".capitalize
+      end
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'There are no people in the library'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, Age: #{person.age}, ID: #{person.id}"
+      end
+    end
+  end
```

Comment:
Well done overall 💯 you're doing great but as mentioned by the previous reviewer, you're mixing up things a bit when showing the data, take a look for the `Student` 👇🏽

![Screenshot 2023-04-22 at 18 53 03](https://user-images.githubusercontent.com/45739963/233810894-223c4ca9-2b8f-4299-89ca-5fdc428f56e6.png)

Age is not displaying the value I added for it ❌  and we have the same problem for the `Teacher`  👇🏽 

![Screenshot 2023-04-22 at 18 54 34](https://user-images.githubusercontent.com/45739963/233810935-067be583-5f43-40d6-b278-1af010c09112.png)

In the teacher's case, `name` displays `age` and `age` displays `name` 🤧